README.md

```diff
@@ -40,6 +40,7 @@ php tao/scripts/taoUpdate.php
 ### Feature Flags
 
 
-| Variable         | Description                                                               | Default value |
-|------------------|---------------------------------------------------------------------------|---------------|
-| FEATURE_FLAG_FLA | Toggles certain media-interaction options' availability in item authoring | false         |
+| Variable                                    | Description                                                               | Default value |
+|---------------------------------------------|---------------------------------------------------------------------------|---------------|
+| FEATURE_FLAG_FLA                            | Toggles certain media-interaction options' availability in item authoring | false         |
```

Comment:
It seems a bit generic this name. FLA is a type of feature, but are we disabling all FLA features with this FF?

Reply:
I'm not touching FLA ff. It was just modification of table here.